Sources/Build/llbuild.swift

```diff
@@ -92,7 +100,7 @@ public struct LLBuildManifestGenerator {
     }
 
     /// Create link command for products.
-    private func createLinkCommand(_ buildProduct: ProductBuildDescription) -> Command {
+    private func createLinkTarget(_ buildProduct: ProductBuildDescription) -> Target {
```

Comment:
Ah, I forgot that we can have multiple products now. This sounds correct I think.